R/estimateDiversity.R

```diff
@@ -239,17 +294,70 @@ setMethod("estimateDiversity", signature = c(x = "SummarizedExperiment"),
     vegan::fisher.alpha(t(mat))
 }
 
+.calc_pd <- function(mat, tree, ...){
+
+    # Gets name of the taxa
+    taxa <- rownames(mat)
+    # Gets name of the samples
+    samples <- colnames(mat)
+
+    # Loops through every sample
+    PDs <- BiocParallel::bplapply(samples, function(sample){
```

Comment:
You have to add the default `BPPARAM = SerialParam()` argument to this function, because the otherwise the system default will be used aka. parallelised by default. See other functions for example implementation, `runJSD` or `runUniFrac` for example